pkg/client/load_balancer.go

```diff
@@ -526,3 +526,22 @@ func (lb *LoadBalancerAPIService) GetSpecificLBVirtualServer(
 
 	return LBVSResp, err
 }
+
+func (lb *LoadBalancerAPIService) GetLBPoolMemberGroup(
+	ctx context.Context,
+	serviceID int,
+) (models.GetMemeberGroupForPool, error) {
+	LBMemberGroupResp := models.GetMemeberGroupForPool{}
+	LBMemberGroupInput := &api{
+		method: "GET",
+		path: fmt.Sprintf("%s/%s/%d/%s",
+			consts.NetworksPath, consts.ServerPath, serviceID, consts.GroupsPath),
```

Comment:
did you review the reference to old path variable which u used for group ?

Reply:
yes i used and referred the edge cluster example

Reply:
LBPoolMemberGroupPath = "networks/servers/1/groups" should be removed

Reply:
i created branch from master so i did not get LBPoolMemberGroupPath here

Reply:
removed in PR https://github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pull/54